bin/lms-rtparser.php

```diff
@@ -227,6 +227,8 @@ function ($value) {
     isset($smtp_options['pass']) ? $smtp_options['pass'] : ConfigHelper::GetConfig('mail.smtp_password')
 );
 $rtparser_use_seen_flag = ConfigHelper::checkValue(ConfigHelper::getConfig('rt.imap_use_seen_flag', true));
+$rtparser_handled_mail_post_action = ConfigHelper::getConfig('rt.rtparser_handled_mail_post_action', 'seen, delete, expunge');
+
```

Comment:
Jaki sens ma podanie domyślnie 3 akcji na raz?

Reply:
Uprościłem - motywacja jest taka, żeby zostawić domyślnie delete&expunge by uniknąć przepełnienia skrzynki.

Reply:
Co wg Ciebie oznacza zapis `seen, delete, expunge` z domyślnej wartości tego ustawienia?

Reply:
Oznacza to, że po zaczytaniu maila przez LMS zostaną wykonane następujące akcje:
seen - oznaczenie maila jako przeczytany,
delete - oznaczenie maila "do usunięcia",
expunge - przeniesienie oznaczonych maili do usunięcia do kosza

Reply:
Jaki sens ma wykonanie wszystkich akcji jednocześnie? :)

Reply:
One nie są wykonywane jednocześnie a w jedynej słusznej kolejności ale w odpowiedniej kolejności.
Pamiętaj, że w koszu możesz też chcieć mieć wiadomości odczytane.

Reply:
Niby mógłbym to rozbić na zmienne:
`rt.rtparser_handled_mail_set_flag` (dotyczy maila)
`rt.rtparser_expunge_before_close` (dotyczy całego połączenia)
Ale nie widzę sensu a Ty widzisz go?

W jednej zmiennej jest to również w pełni konfigurowalne i nie ma ograniczeń.